### ISSUE TITLE
Unable to train custom entity

### ISSUE THREAD

**Issue by xanish** [explosion/spaCy#709]
I was trying to create my own entity using this [example](https://github.com/explosion/spaCy/blob/master/examples/training/train_ner.py).

```
Traceback (most recent call last):
  File "train_ner.py", line 91, in <module>
    main('ner')
  File "train_ner.py", line 47, in main
    nlp = spacy.load('en', parser=False, entity=False, add_vectors=False)
  File "/usr/local/lib/python3.5/dist-packages/spacy/__init__.py", line 38, in load
    return cls(**overrides)
  File "/usr/local/lib/python3.5/dist-packages/spacy/en/__init__.py", line 33, in __init__
    overrides = _fix_deprecated_glove_vectors_loading(overrides)
  File "/usr/local/lib/python3.5/dist-packages/spacy/en/__init__.py", line 57, in _fix_deprecated_glove_vectors_loading
    if vec_path is not None:
UnboundLocalError: local variable 'vec_path' referenced before assignment
```

The above error points to this [file](https://github.com/explosion/spaCy/blob/master/spacy/en/__init__.py). And also is there a way to use the inbuilt entities with my own custom entity without training it all over again. Thanks.

## Your Environment

* **Operating System: Ubuntu 16.04** 
* **Python Version Used: 3.5.2** 
* **spaCy Version Used: 1.4.0**

**Comment by honnibal** [comment]
Hey,

This should be fixed in 1.5.0 — could you update and try again? Please reopen if the problem reoccurs

Overall the experience of training new entity models is still not wonderful, unfortunately. We're working on this, but it's likely you'll hit another issue. Please comment on the relevant issue if there's already one open that looks like the problem you're experiencing; or open a new one if there's nothing appropriate. The [Gitter chat](https://gitter.im/explosion/spaCy) is another likely source of help.

**Comment by xanish** [comment]
Will check at the earliest possible, Thanks!
